modules/web-console-cso.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * web_console/scan-from-web-console.adoc
+
+[id="web-console-cso_{context}"]
+= Run Container Security Operator in {product-title}
+
+To start the Container Security Operator, go to the {product-title}
+web console, then select and install that Operator from the Operator Hub,
+as described here.
+
+.Prerequisites
+
+* Be running {product-title} 4.2 or later
+* Have administrator privileges to the {product-title} cluster
+* Be running containers that come from a Red Hat Quay or Quay.io registry
+
+.Procedure
+
+To start using the CSO in OpenShift, do the following:
```

Comment:
I'd remove this, we typically don't use an intro sentence before the steps.

Reply:
Done.